build.gradle.kts

```
@@ -52,6 +56,7 @@ buildscript {
 
 plugins {
     java
+    kotlin("jvm") version "1.4.21"
```

Let's use the same approach we already use for the other plugins.

```suggestion
    kotlin("jvm").version("1.4.21")
```

Also, can we rely on the `Deps` object for the version?

---

It's a bit easier to read with spaces instead of dots and braces as delimiters. So, I'd suggest doing the other way around.